samples/crypto/persistent_key_usage/README.rst

```diff
@@ -1,36 +1,34 @@
 .. _crypto_persistent_key:
 
-Crypto: AES CTR
-###############
+Crypto: Persistent key storage
+##############################
 
 .. contents::
    :local:
    :depth: 2
 
+The persistent key sample shows how to generate a persistent key using the Platform Security Architecture (PSA) APIs.
+Persistent keys are stored in the Internal Trusted Storage (ITS) of the device and retain their value between resets.
+A persistent key becomes unusable when the ``psa_destroy_key`` function is called.
+
 Overview
 ********
 
-The persistent key sample shows how to generate a persistent key using the PSA APIs. Persistent keys
-are stored in the Internal Trusted Storage (ITS) of the device and retain their value between resets.
-A persistent key becomes unusuable when the psa_destroy_key function is called.
-
-In this sample we create an AES 128-bit key (CTR block cipher mode) for demonstration purposes.
+In this sample, an AES 128-bit key is created (CTR block cipher mode) for demonstration purposes.
```

Comment:
I think "created" can be ended (full stop). The additional text in this line is somewhat superfluous...